device/crypto.go

```diff
@@ -142,7 +133,7 @@ func (d *Device) ensureCSR() error {
 
 	privateKey, ok := parsedKey.(*rsa.PrivateKey)
 	if !ok {
-		return errors.New("Unable to parse RSA private key, clearing the crypto store")
+		return errors.New("unable to parse RSA private key, clearing the crypto store")
```

Comment:
Shouldn't this be rebased following the fact that now the SDK generates ECDSA keys?